parsec/core/fs/workspacefs/workspacefile.py

```diff
@@ -33,13 +33,13 @@ class WorkspaceFile:
     Keyword arguments:
     transactions -- object to use for file transactions.
     path -- relative path of the file.
-    mode -- the mode where the file have been opened. It needs to have exactly one of "awrx".
+    mode -- the mode where the file have been opened. It needs to have exactly one of "arwx".
```

Comment:
Is it a false positive? I think here `r`, `w` and `x` stands for read, write and execute.

Reply:
Yes it is, it's just the a prefer the word order `read write execute`